compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -1200,6 +1221,8 @@ fn build_union_type_di_node<'ll, 'tcx>(
             Stub::Union,
             unique_type_id,
             &type_name,
+            unknown_file_metadata(cx),
+            UNKNOWN_LINE_NUMBER,
```

Comment:
`union`s should also have line numbers (you can call the extracted function I mentioned above here)